### PR TITLE
Update and rename xx.json to de.json

### PR DESCRIPTION
**Pull request by Taranuka**
german translation.

### CODE REVIEW

**Comment by IceOfWraith** [comment]
I think you accidentally named the file en.json instead of de.json. :)

**Comment by IceOfWraith** [comment]
Lol! Disregard... My browser translated that automatically...

File: de.json [hunk 1]
```diff
@@ -0,0 +1,260 @@
...
+    "AMP Theme": "AMP-Design",
+    "Offset in seconds to advance or delay execution of scheduled tasks that use time-based triggers.": "Offset in Sekunden, um die Ausführung geplanter Aufgaben, die zeitbasierte Trigger verwenden, vorzuziehen oder zu verzögern.",
+    "What AMP should do when it starts": "Was AMP tun soll, wenn es startet",
+    "Affects all users - change AMPs visual appearance with different themes. You will need to do a force-reload for this to take effect.": "Betrifft alle Benutzer – Ändere das Erscheinungsbild von AMP mit verschiedenen Designs. Sie müssen die Seite neuladenh, damit dies wirksam wird.",
+    "External Services": "Externe Dienste",
+    "Use SSL for SMTP": "SSL für SMTP verwenden",
+    "SMTP Server Port": "SMTP-Server-Port",
+    "SMTP Password": "SMTP-Passwort",
+    "Pushbullet Access Token": "Pushbullet-Zugriffstoken",
+    "SMTP Server Address": "SMTP-Serveradresse",
```
Comment (BroOtti) [inline de.json:185]: SMTP Server

File: de.json [hunk 2]
```diff
@@ -0,0 +1,260 @@
...
+    "SMTP Username": "SMTP-Benutzername",
```
Comment (BroOtti) [inline de.json:186]: SMTP Benutzername

File: de.json [hunk 3]
```diff
@@ -0,0 +1,260 @@
...
+    "Email 'From' address": "E-Mail-Absenderadresse",
```
Comment (BroOtti) [inline de.json:187]: SMTP Absenderadresse

File: de.json [hunk 4]
```diff
@@ -0,0 +1,260 @@
...
+    "Send Test Message": "Testnachricht senden",
+    "The address used for outgoing emails.": "Die für ausgehende E-Mails verwendete Adresse.",
+    "Stored Backups": "Gespeicherte Sicherungen",
```
Comment (BroOtti) [inline de.json:190]: Vorhandende Sicherungen

File: de.json [hunk 5]
```diff
@@ -0,0 +1,260 @@
...
+    "Name": "Name",
+    "Taken By": "Genommen von",
```
Comment (BroOtti) [inline de.json:192]: Erstellt durch

File: de.json [hunk 6]
```diff
@@ -0,0 +1,260 @@
...
+    "Time Stamp": "Zeitstempel",
+    "Size": "Größe",
+    "Create a Backup": "Sicherung erstellen",
+    "Backup Name": "Sicherungsname",
+    "A short, unique name for your backup": "Ein kurzer, eindeutiger Name der Sicherung",
+    "A description of the backup and why you're taking it (optional)": "Eine Beschreibung der Sicherung und warum Sie es nehmen (optional)",
```
Comment (BroOtti) [inline de.json:198]: Eine Beschreibung der Sicherung und warum du sie erstellst (optional)

File: de.json [hunk 7]
```diff
@@ -0,0 +1,260 @@
...
+    "Mark as sticky": "Als wichtig markieren",
+    "Sticky backups won't be deleted to make room for automatic backups": "Wichtig markierte Sicherungen werden nicht gelöscht, um Platz für automatische Sicherungen zu schaffen",
+    "Create Backup": "Eine Sicherung erstellen",
```
Comment (BroOtti) [inline de.json:201]: Sicherung erstellen

File: de.json [hunk 8]
```diff
@@ -0,0 +1,260 @@
...
+    "Notifications": "Benachrichtigungen",
+    "Delete Backup": "Sicherung löschen",
+    "Restore Backup": "Sicherung wiederherstellen",
+    "Confirm Deletion": "Löschen bestätigen",
+    "Are you sure you wish to delete this backup?": "Möchten Sie diese Sicherung wirklich löschen?",
```
Comment (BroOtti) [inline de.json:206]: Bist du sicher, dass du diese Sicherung löschen möchtest?

File: de.json [hunk 9]
```diff
@@ -0,0 +1,260 @@
...
+    "Creating Backup": "Sicherung wird erstellt",
+    "Confirm Restore": "Wiederherstellung bestätigen",
+    "Are you sure you wish to restore this backup? Any files in this archive will overwrite any with the same name that already exist.": "Möchten Sie diese Sicherung wirklich wiederherstellen? Alle Dateien in diesem Archiv überschreiben alle bereits vorhandenen mit demselben Namen.",
```
Comment (BroOtti) [inline de.json:209]: Bist du sicher, dass du diese Sicherung wiederherstellen möchtest? Jede Datei dieses Archivs wird bereits existierende Dateien überschreiben.

File: de.json [hunk 10]
```diff
@@ -0,0 +1,260 @@
...
+    "Copy": "Kopieren",
+    "Paste Here": "Einfügen",
+    "Rename": "Umbenennen",
+    "Exclude from backups": "Von Sicherungen ausschließen",
+    "Download file here": "Datei herunterladen",
```
Comment (BroOtti) [inline de.json:214]: Datei hierhin herunterladen

File: de.json [hunk 11]
```diff
@@ -0,0 +1,260 @@
...
+    "Last Modified": "Zuletzt bearbeitet",
+    "Editable": "Bearbeitbar",
+    "Downloadable": "Herunterladbar",
+    "Full Path": "Vollständiger Pfad",
+    "Excluded from backups": "Von Sicherungen ausgeschlossen",
+    "Download": "Herunterladen",
+    "Calculate md5 hash": "MD5-Hash berechnen",
+    "Edit": "Bearbeiten",
+    "Move to trash": "Löschen",
+    "Are you sure you want to move the selected item to the trash?": "Möchten Sie das ausgewählte Element wirklich in den Papierkorb verschieben?",
```
Comment (BroOtti) [inline de.json:228]: Bist du sicher, dass du die ausgewählte Datei löschen möchtest?

File: de.json [hunk 12]
```diff
@@ -0,0 +1,260 @@
...
+    "Delete Item": "Eigenschaft löschen",
```
Comment (BroOtti) [inline de.json:229]: Datei löschen

File: de.json [hunk 13]
```diff
@@ -0,0 +1,260 @@
...
+    "Rename Directory": "Ordner umbenennen",
+    "Please enter a new directory name": "Bitte geben Sie einen neuen Verzeichnisnamen ein",
```
Comment (BroOtti) [inline de.json:231]: Bitte gebe einen neuen Ordnernamen ein

File: de.json [hunk 14]
```diff
@@ -0,0 +1,260 @@
...
+    "Rename File": "Datei umbenennen",
+    "Please enter a new filename": "Proszę wprowadzić nową nazwę pliku",
```
Comment (BroOtti) [inline de.json:233]: Bitte gebe einen neuen Dateinamen ein

File: de.json [hunk 15]
```diff
@@ -0,0 +1,260 @@
...
+    "New Directory": "Neuer Ordner",
+    "Please enter a name for the new directory": "Bitte geben Sie einen Namen für den neuen Ordner ein",
```
Comment (BroOtti) [inline de.json:235]: Bitte gebe einen Namen für den neuen Ordner ein

File: de.json [hunk 16]
```diff
@@ -0,0 +1,260 @@
...
+    "No files to paste": "Keine Dateien zum Einfügen",
+    "You have no files in the clipboard to paste": "Sie haben keine Dateien in der Zwischenablage zum Einfügen",
```
Comment (BroOtti) [inline de.json:237]: Du hast keine Dateien in der Zwischenablage zum Einfügen

File: de.json [hunk 17]
```diff
@@ -0,0 +1,260 @@
...
+    "Want help fast?": "Brauchen Sie schnelle Hilfe?",
```
Comment (BroOtti) [inline de.json:238]: Willst du schnell Hilfe erhalten?

File: de.json [hunk 18]
```diff
@@ -0,0 +1,260 @@
...
+    "Join CubeCoders on Discord!": "Tritt CubeCoders auf Discord bei!",
+    "Otherwise, for support with AMP please visit the": "Andernfalls besuchen Sie für Hilfe mit AMP bitte das ",
```
Comment (BroOtti) [inline de.json:240]: Andernfalls besuche für Hilfe mit AMP bitte das

File: de.json [hunk 19]
```diff
@@ -0,0 +1,260 @@
...
+    "support board": "Hilfe-Forum",
+    "Always make sure to include the information listed below.": "Achten Sie immer darauf, die unten aufgeführten Informationen anzugeben.",
```
Comment (BroOtti) [inline de.json:242]: Achte immer darauf folgende Informationen anzugeben.

File: de.json [hunk 20]
```diff
@@ -0,0 +1,260 @@
...
+    "Currently running version": "Derzeit laufende Version",
+    "View Latest Release Notes": "Neueste Versionshistorie",
+    "Diagnostics Data": "Diagnosedaten",
+    "Platform": "Plattform",
+    "System Type": "Systemtyp",
+    "Virtualization": "Virtualisierung",
+    "Loaded Plugins": "Geladene Plugins",
+    "Application Name": "Anwendungsname",
+    "Application Version": "Anwendungsversion",
+    "Restart this AMP instance": "Diese AMP-Instant neustarten",
```
Comment (BroOtti) [inline de.json:253]: Diese AMP Instanz neustarten

File: de.json [hunk 21]
```diff
@@ -0,0 +1,260 @@
...
+    "Upgrade this AMP instance": "Diese AMP-Instanz aktualisieren",
```
Comment (BroOtti) [inline de.json:254]: Diese AMP Instanz aktualisieren

File: de.json [hunk 22]
```diff
@@ -0,0 +1,260 @@
...
+    "Confirm Upgrade": "Aktualisierung bestätigen",
+    "Upgrading this instance may take up to 5 minutes. You will not be able to log into any instances that depend on this ADS instance.": "Die Aktualisierung dieser Instanz kann bis zu 5 Minuten dauern. Sie können sich nicht bei Instanzen anmelden, die von dieser ADS-Instanz abhängen.",
```
Comment (BroOtti) [inline de.json:256]: Die Aktualisierung dieser Instanz kann bis zu 5 Minuten dauern. Du kannst dich in keine Instanz anmelden, die von dieser ADS Instanz abhängig ist.

File: de.json [hunk 23]
```diff
@@ -0,0 +1,260 @@
...
+    "Keep Running": "gestartet",
```
Comment (BroOtti) [inline de.json:257]: Instanz gestartet lassen

File: de.json [hunk 24]
```diff
@@ -0,0 +1,260 @@
...
+    "Upgrade Now": "Jetzt upgraden"
```
Comment (BroOtti) [inline de.json:258]: Jetzt aktualisieren

**Review by BroOtti — CHANGES_REQUESTED** [review]
Suggested some changes

**Comment by BroOtti** [comment]
Just to mention it. There are still many translations missing. At least I enabled the translator mode and got a file with over 500 lines. I also haven't checked if the translations fit in their place :)

File: de.json [hunk 25]
```diff
@@ -0,0 +1,260 @@
...
+    "Add new trigger": "Neuen Trigger hinzufügen",
+    "User Management": "Benutzerverwaltung",
+    "Create User": "Benutzer erstellen",
+    "New Username": "Neuer Benutzername",
+    "Please enter the login username for the new user": "Bitte geben Sie den Login-Benutzernamen für den neuen Benutzer ein",
+    "Add a new trigger": "Trigger hinzufügen",
+    "What type of trigger would you like to add?": "Welchen Trigger möchten Sie hinzufügen?",
+    "Event Trigger": "Ereignis-Trigger",
+    "For responding to events raised by in-application actions or by other AMP components": "Zum Reagieren auf Ereignisse, die durch In-App-Aktionen oder andere AMP-Komponenten ausgelöst werden",
+    "Simple Time Interval": "Einfaches Zeitintervall",
```
[the remark this answers is not in the document]
Comment (gOOvER) [reply de.json:106]: Intervall ist aber nicht wirklich ein Zeitplan

**Comment by gOOvER** [comment]
Mike should be add the translations into a Translationssystem like Weblate

**Review by Taranuka — COMMENTED** [review]
top

**Comment by Taranuka** [comment]
> > top
> 
> Did you just mark all my suggested changes without any changes or discussion? Just to note, some of my suggestions need to be discussed and maybe just personal flavor. Some suggestions are for a better wording and others are for a unified wording. So just marking everything as resolved without any changes won't lead to an approval from me. PS: You also mistyped one word in the translation :) So at least that needs to be fixed before somebody should approve the request :D

Sorry. It is my first try on GitHub. Actually I dont know to use most features. Probably you could help?

 Your suggestions are fine. How I take over your suggestions without to edit 60+ lines?

For most translations I used Google translate. Some I have changed for a more approbiate meaning.

**Comment by BroOtti** [comment]
> Sorry. It is my first try on GitHub. Actually I dont know to use most features. Probably you could help?
> 
> Your suggestions are fine. How I take over your suggestions without to edit 60+ lines?
> 
> For most translations I used Google translate. Some I have changed for a more approbiate meaning.

No prob :D It's also my first try on GitHub ^^ So I don't know if there's a better way to do it. I think you need to edit the file yourself and create a new commit. Just go comment for comment :) Best would be to reply to my comments and then after you changed it or we discussed it marking it as resolved.
Also, feel free to ask me why I suggested some of the changes :) I'm not perfect :D There could be still better translations :)

File: de.json (hunk 25 above)
[the remark this answers is not in the document]
Comment (BroOtti) [reply de.json:106]: Ja, stimmt schon :D Dann belassen wir das bei Intervall :)

**Comment by BroOtti** [comment]
I installed a Weblate instance to make it easier for us :)
https://weblate.bromunity.de/projects/amp-translations/github/
I generated my own en.json file and imported your PR to the german translation. Not all strings are included in my generated en.json. Feel free to update my en.json template inside https://github.com/BroOtti/AMPTranslations

**Review by PhonicUK — APPROVED** [review]

**Review by PhonicUK — COMMENTED** [review]
What do you make of this feedback from ChatGPT?

Overall, the translations look solid, but there are some inconsistencies, typographical issues, and places where the translation could be improved. Here's a detailed list:

Months and Days Abbreviation:
---

* The abbreviations for months and days of the week are not commonly used in German. You can keep the full name of the months, and for the short days of the week, you can consider using the first two letters (e.g., "Mo", "Di", etc.).

Typographical Mistakes:
---

* "beim booten starten" should be "beim Booten starten" with a capital "B".
* "Instanz und die Anwenndung starten" has a typo, it should be "Instanz und die Anwendung starten".
* "Instanz beim booten starten" should be "Instanz beim Booten starten".

Inconsistencies:
---

* You've used "Aktualisieren" for the "Update" and "Refresh" keys. While not incorrect, you might want to differentiate them, e.g., "Update" could be "Aktualisieren" and "Refresh" could be "Erneuern" or "Neu laden."

Grammatical and Stylistic Improvements:

* "nichts machen" could be more naturally translated as "Nichts tun."
* "Kann nicht gestartet werden" might be better as "Kann nicht ausgeführt werden."
* "Bestätigen Sie das Herunterfahren" might be better as "Herunterfahren bestätigen."

Potentially Confusing Translations:
---

* "Sicherung" for "Backups" is technically correct, but "Backups" is also commonly used in German IT context and might be more recognizable to users.
* "Speichernutzung" for "Memory Usage" is correct, but "Arbeitsspeichernutzung" might be more precise.
* "CPU auslastung" should be "CPU-Auslastung" (with a hyphen).
* "Bestätigen Sie das Herunterfahren" is a bit formal. Consider using "Herunterfahren bestätigen."

Other Considerations:
---

* There are some technical terms that could be left in English depending on the target audience, such as "Endpoint", "Instance", "Docker Container", etc. Consider the context and the users' familiarity with these terms in German.
* There's an inconsistency with "Instanz beenden" and "Instanz starten" versus "Instanz löschen" and "Instanz erstellen". Consider using "Instanz stoppen" instead of "Instanz beenden" to maintain uniformity.
* There might be more specific translations for some terms based on the context of your application. If this is a specialized software, the users might expect certain terminology.
* Please update the translations based on these suggestions to ensure accuracy and consistency.

**Comment by BroOtti** [comment]
Most of the feedback from ChatGPT was already in my change suggestions :D But this Pull Request here is a bit out of date. Pull Request #6 is a more complete version and also used this PR as base. PR #6 was made with the help of my weblate instance and includes contributions from multiple people at Discord.

**Review by PhonicUK — APPROVED** [review]

**Comment by IceOfWraith** [comment]
Just a heads up, this is an incomplete translation due to the recent changes added to AMP to allow pulling more. I'll find the code snippet Mike provided to grab more. But even then, there isn't an easy way to maintain this as more strings are added to the AMP GUI.

**Pull request merged**